index.html

```diff
@@ -0,0 +1,236 @@
+        <p class="line1">TAWA // TETRÁFONO ENSAMBLE // CHANGUITAS (Arg.)</p>
+        <p class="line2">TAMBORES Y OTROS DEMONIOS SAMY // GRUPO SENDERO // ALEX ALVEAR</p>
+        <p class="line3">IRAIZ OVIEDO Y SU MISTURA // MARIA TEJADA // TALLER LA BOLA // JATARI</p>
+        <p class="line4">WAÑUKTA TONIC // KARINA CLAVIJO // JAYAC // DANIEL MANCERO // PITEKUS</p>
+      </div>
+
+      <div class="date-venue">
+        <p class="date mx-4">2023.06.22(THU)</p>
+        <p class="venue mx-4">@Ciudad Mitad del Mundo, Central Plaza</p>
+      </div>
+    </section>
+
+
+    <!-- MAIN PROGRAM SECTION -->
+    <section class="info">
+
+      <div class="title pt-5
+      ">
+        <h2 class="mp-title d-flex justify-content-center">Main Program</h2>
+      </div>
+
+      <!---------------------------------------- MAIN PROGRAM CARDS-->
+      <div class="main-program gap-3 mx-auto  bg-ligh">
+
+        <div class="program-cards ">
+            <img src="./capstoneimgs/ticketicon.svg" alt="tickets icon">
+            <h3><a href="#" class="program-cards-title">Tickets</h3></a>
+            <p class="pc-text m-2">Here you will find more information about obtaining tickets to the Inti Raymi Music Festival. </p>
+          </a>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/scheduleicon.svg" alt="schedule icon">
+          <h3><a href="#" class="program-cards-title">Schedule</h3></a>
+          <p class="pc-text m-2">The Festival will take place on June the 22nd, 2023, starting at noon. See inside for more information.</p>
+        </a>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/servicesicon.svg" alt="food and services icon">
+          <h3><a href="#" class="program-cards-title">Food & Services</a></h3>
+          <p>You will have a variety of services available like food and souvenir stands. See inside
+            for more details</p>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/exibitionicon.svg" alt="Exhibition icon">
+          <h3><a href="#" class="program-cards-title">Inti Raymi Exhibition</a></h3>
+          <p>Join the exhibition to learn more about this incredible celebration. It is for all public that attend the
+            festival!</p>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/programicon.svg" alt="Forum icon">
+          <h3><a href="#" class="program-cards-title">Forum</a></h3>
+          <p>Foreigners and nationals are welcome in this festival! Get in touch with other people interested in this event.</p>
+        </div>
+
+        <button class="join-inti">Join Inti Raymi Celebration 2023</button>
+      </div>
+
+      <div class="desktop-see d-none d-md-flex justify-content-center py-5 mt-5">
+        <h4 href="#" class="desktop-see-program">SEE THE WHOLE PROGRAM</h4>
+      </div>
+
+    </section>
```

Comment:
- [ ] For the main program section, please consider reducing the font size, adding some gaps between each card to differentiate them, and adding some spacing between the edge of the screen and each card! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/226498703-5af83fe4-0c26-4f33-99e0-b4f4cbb3af23.png) | ![image](https://user-images.githubusercontent.com/77069011/226498595-fe03cbf7-8351-4d6e-92a2-b46096f49dc8.png) |

Reply:
Of course, it's done!

Reply:
- [ ] This issue is well resolved :heavy_check_mark: